.github/workflows/full.yml

```diff
@@ -22,6 +22,6 @@ jobs:
           NODE_OPTIONS: '--max-old-space-size=6144'
           FONTAWESOME_NPM_AUTH_TOKEN: ${{ secrets.FONTAWESOME_NPM_AUTH_TOKEN }}
         run: |
-          sbt stage
+          sbt -v -J-Xmx6g stage
```

Comment:
Unrelated change, but CI builds were failing because of OOM.

Reply:
Maybe because of the addition of http4s?

Reply:
It was already happening. See for example https://github.com/gemini-hlsw/explore/pull/1060. We actually increased the memory a while ago for the Heroku build but forgot to do it here too.

Reply:
Ah #1060 already has http4s-core (but not http4s-client), so maybe?